iphone/Classes/TiUIiOSProxy.m

```diff
@@ -715,6 +719,13 @@ -(id)createBlurView:(id)args
 }
 #endif
 
+#ifdef USE_TI_UIIOSSTEPPER
+-(id)createStepper:(id)args
+{
+    return [[[TiUIiOSStepperProxy alloc] _initWithPageContext:[self executionContext] args:args] autorelease];
```

Comment:
Why not `[self pageContext]`? I remember we should use that for newer API's, but they should work the same.